ukis_pysat/raster.py

```diff
@@ -360,13 +361,17 @@ def to_dask_array(self, chunk_size=(1, 6000, 6000)):
         self.da_arr = da.from_array(self.__arr, chunks=chunk_size)
         return self.da_arr
 
-    def write_to_file(self, path_to_file, dtype=rasterio.uint16, driver="GTiff"):
+    def write_to_file(self, path_to_file, dtype, driver="GTiff", compress=None):
         """
         Write a dataset to file.
         :param path_to_file: str, path to new file
-        :param dtype: datatype, optional (default: rasterio.uint16)
+        :param dtype: datatype, like rasterio.uint16, 'float32' or 'min' to use the minimum type to represent values

```

Comment:
can you please verify if we can also use numpy dtypes here?

Reply:
```suggestion
        :param dtype: datatype, like np.uint16, 'float32' or 'min' to use the minimum type to represent values

```

We can, numpy also works with [strings](https://numpy.org/doc/1.18/reference/arrays.dtypes.html), like in the last example here. I agree, that it might be used more widely in our environment.